sigle/src/modules/analytics/stats/StatsFrame.tsx

```diff
@@ -79,13 +85,47 @@ export const StatsFrame = () => {
       };
     });
 
+    const viewTotal = statsData.historical
+      .map((item) => item.pageviews)
+      .reduce(function (previousValue, currentValue) {
+        return previousValue + currentValue;
+      }, 0);
+
+    const visitTotal = statsData.historical
+      .map((item) => item.visits)
+      .reduce(function (previousValue, currentValue) {
+        return previousValue + currentValue;
+      }, 0);
+
```

Comment:
You can probably do just one reduce for this (also without the map) for better perf :) 
smth like: 

```
const viewTotal = statsData.historical
      .reduce(function (previousValue, currentValue) {
        return { pageviews: previousValue.pageviews + currentValue.pageviews }
      }, {pageviews: 0, visits: 0});
```